src/config/settings.js

```diff
@@ -91,6 +91,19 @@ function init() {
   return settings;
 }
 
+async function isFirstRun() {
+  let settingsPath = join(paths.config(), 'settings.json');
+  try {
+    await util.promisify(fs.access, fs.constants.F_OK)(settingsPath, fs.constants.F_OK);
+    return false;
+  } catch(err) {
+    if (err.code !== 'ENOENT') {
+      console.log(`Checking for existence of ${settingsPath}, got error ${err}`);
```

Comment:
Should we throw here?

Reply:
I don't think it's consequential enough.